client/parser/tinymembench_parser.py

```diff
@@ -24,48 +24,25 @@ def tinyResult(content, outfp):
 	dic['tiny_latency']['Hugepage dual random read'] = []
```

Comment:
Please remove space with "_" to make the test name convention uniform and also to parse the results easily to excel format.